kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/iptables/PortForwardRule.java

```diff
@@ -384,7 +391,10 @@ public void setPermittedMAC(String permittedMAC) {
      * @return the sourcePortRange
      */
     public String getSourcePortRange() {
-        return this.sourcePortRange;
+        if (this.sourcePortStart == this.sourcePortEnd && this.sourcePortStart == 0) {
+            return null;
+        }
+        return this.sourcePortStart + ":" + this.sourcePortEnd;
     }
```

Comment:
So, if there is only one source port, the result is port:port (i.e. 10100:10100). Is it correct?

Reply:
yes